passage/src/main/java/id/passage/android/Passage.kt

```diff
@@ -654,4 +656,57 @@ public final class Passage(
         tokenStore.setTokens(authResult)
     }
     // endregion
+
+    // region OIDC Methods
+
+    /**
+     * Authentication Method for Hosted Apps
+     *
+     * If your Passage app is Hosted, use this method to register and log in your user.
+     * This method will open up a Passage login experience on a Chrome tab.
+     */
+    public fun hostedAuthStart() {
+        PassageHosted.openChromeTab(
+            activity,
+        )
+    }
+
+    /**
+     * Finish Hosted Auth for Hosted Apps
+     *
+     * This method completes the hosted authentication process by exchanging the provided authorization code for Passage tokens.
+     *
+     * @param code The code returned from app link redirect to your activity.
+     * @param clientSecret You hosted app's client secret, found in Passage Console's OIDC Settings.
+     * @param state The state returned from app link redirect to your activity.
+     * @throws HostedAuthorizationError
+     */
+
+    suspend fun hostedAuthFinish(
+        code: String,
+        clientSecret: String,
+        state: String,
+    ) {
```

Comment:
Just to match our other existing auth methods, let's return the `PassageAuthResult` here for now. We can reconsider having the auth methods return `void` in 2.0.
Additionally, we'll need to return the `idToken` here for users who opt out of using the built in token store (more on this in a later comment below).
```suggestion
    ): Pair(PassageAuthResult, String) {
```